### DISCUSSION TITLE
Error: Failure while executing;

### DISCUSSION THREAD

**Discussion by vascoferraz** [Homebrew/discussions#2589] [Everyday usage]
Hey guys,

When I execute this command: **brew style --fix** I get this error message:

**Error: Failure while executing; `patch --get=0 --force --directory=/ --strip=0` exited with 2. Here's the output:**

![screenshot](https://user-images.githubusercontent.com/10391988/144683741-07c4886b-9f68-4d52-b3f5-898919e9aa80.png)

Any ideas on how to solve this?

**Comment by robertleeblairjr** [comment]
Commenting to confirm that this has the same error code for me that wasn't previously existing. However, I have no idea when it was introduced. But, I don't recall it prior to 3.x.
`brew config`

"HOMEBREW_VERSION: 3.3.9-103-ga0ae4a7
ORIGIN: https://github.com/Homebrew/brew
HEAD: a0ae4a7a02d9b6d886fb99a23ddbe07af7979fb6
Last commit: 7 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 9b79f47c8370c7449aa1470a0c3eb7386e8d5f3d
Core tap last commit: 5 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.YxWpAFjL9i/org.xquartz:0
HOMEBREW_EDITOR: /usr/local/bin/nano
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 13.0.0 build 1300
Git: 2.34.1 => /usr/local/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.1-x86_64
CLT: 13.2.0.0.1.1638488800
Xcode: 13.2.1"

**Comment by Arvind-Srinivasan** [comment]
Also have the issue.

**Comment by vascoferraz** [comment]
I did nothing (just the regular updates) and the issue went away.
